lib/pages/bus/bus_rule_page.dart

```diff
@@ -42,106 +42,93 @@ class BusRulePageState extends State<BusRulePage> {
         child: SelectableText.rich(
           TextSpan(
               style: TextStyle(
-                  color: ApTheme.of(context).grey, height: 1.3, fontSize: 16.0),
```

Comment:
Please pull origin to your fork repository.
Your code is very old version, not work in latest master version.